tests/integration/testdata/diff/004-no-diff-plugin/protocol-initial-order-plugins.yaml

```diff
@@ -13,7 +13,6 @@ services:
     - grpc
     - https
   - name: prometheus
-    service: mockbin
```

Comment:
@Prashansa-K why was this change needed 🤔  - iirc these files were used for plugin protocols ordering fix